### DISCUSSION TITLE
lavfi-filter: reliably getting the external-file

### DISCUSSION THREAD

**Discussion by skinkie** [mpv-player/mpv#14708] [Q&A]
I am overlaying a program logo on top of a video. I am using --external-file=/tmp/proglogo.png --lavfi-complex="[vid1][vid2]overlay=main_w-270:50[vo]" for this effort.

Recently I received a video that contains an extra mjpeg frame, a thumbnail. This causes vid2 to be the _thumbnail_, instead of the _external-file_.

How can you reliably get the external-file in lavfi-filter?

[output.txt](https://github.com/mpv-player/mpv/files/6987815/output.txt)

Discussion does not reference pull request
https://github.com/mpv-player/mpv/pull/9122

**Comment by skinkie** [comment]
@kasper93 what commit did complete it?

**Comment by kasper93** [comment]
> what commit did complete it?

The issue tracker is not for questions or discussions. Also questions are not generally answered by commits.

**Reply by skinkie** [reply]
You close an issue which has a pull request. The question why you have closed it as completed is quite normal, if no commit was provided.

**Reply by kasper93** [reply]
I've closed like 100 questions, sorry if you were affected by this change. Your pull request is still open.

**Reply by skinkie** [reply]
Ah, now I understand it. You closed the issues that were _questions_.
